client/signup/processing-screen/index.jsx

```diff
@@ -185,35 +184,35 @@ export class SignupProcessingScreen extends Component {
 	}
 
 	renderUpgradeNudge() {
-		/* Do NOT translate the strings in this function until the abtest is finished. */
+		const { translate } = this.props;
 
 		/* eslint-disable max-len, wpcalypso/jsx-classname-namespace */
 		return (
 			<div className="signup-pricessing__upgrade-nudge">
-				<p className="signup-pricessing__title-subdomain">Your subdomain</p>
+				<p className="signup-pricessing__title-subdomain">{ translate( 'Your subdomain' ) }</p>
 				<div className="signup-pricessing__address-bar">
 					<Gridicon icon="refresh" size={ 24 } />
 					<Gridicon icon="house" size={ 24 } />
 					<p className={ classnames( 'signup-pricessing__address-field', { 'is-placeholder': ! this.state.siteSlug } ) }>{ this.state.siteSlug }</p>
 				</div>
 				<div className="signup-pricessing__bubble">
 					<svg className="signup-pricessing__bubble-tail" viewBox="0 0 47 31" xmlns="http://www.w3.org/2000/svg"><path d="M.261 30.428S14.931 6.646 46.066.528l-11.852 29.9H.26z" fillRule="evenodd" /></svg>
-					<p>Search engines like Google or Bing prefer websites with their own web address and place them higher in search results.</p>
+					<p>{ translate( 'Search engines like Google or Bing prefer websites with their own web address and place them higher in search results.' ) }</p>
 				</div>
 				<p className="signup-pricessing__nudge-message">
-					Looks like your new online home doesn't have its own domain name.
+					{ translate( 'Looks like your new online home doesn\'t have its own domain name.' ) }
 				</p>
-				<Button disabled={ ! this.props.loginHandler } className="signup-pricessing__upgrade-button" onClick={ this.handleClickUpgradeButton }>Upgrade Plan & Get A Domain</Button>
+				<Button disabled={ ! this.props.loginHandler } className="signup-pricessing__upgrade-button" onClick={ this.handleClickUpgradeButton }>{ translate( 'Upgrade Plan & Get A Domain' ) }</Button>
 			</div>
 		);
 		/* eslint-disable max-len, wpcalypso/jsx-classname-namespace */
 	}
 
 	renderUpgradeScreen() {
-		/* Do NOT translate the strings in this function until the abtest is finished. */
+		const { translate } = this.props;
 		const title = this.props.loginHandler
-			? 'Congratulations! Your site is live.'
-			: 'Congratulations! Your website is almost ready.';
+			? translate( 'Congratulations! Your site is live.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Congratulations! WordAds is active on your site.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👎